docs/concepts-guide.md

```diff
@@ -13,6 +13,14 @@ import TabItem from '@theme/TabItem';
 
 This guide is meant to be a comprehensive overview of Temporal concepts.
 
+:::info WORK IN PROGRESS
+
+This guide is a work in progress.
+Some sections may be incomplete.
+Information may change at any time.
+
+:::
+
 ## Temporal
```

Comment:
This H2 doesn't appear in the right nav (internal TOC). Any idea why? (I've seen this behavior sporadically elsewhere, such as in the tctl ref.)

Reply:
![Screen Shot 2022-04-26 at 2 13 56 PM](https://user-images.githubusercontent.com/34380806/165384428-93a10e68-62d4-4539-a666-3e6e448efc22.png)

It is showing up for me in the preview